README.md

```diff
@@ -79,6 +81,10 @@ Feature: MySQL services are not affected due to node failures.
     And verify MySQL-client can still access data.
 ```
 
+<p align="center">
```

Comment:
This video looks good!

Can you move this to a separate section -- named "Demo : <title...>". Place this section before .. the section for "Running a specific Test"